src/organisms/cards/PolicyCard/policy_card.module.scss

```diff
@@ -0,0 +1,127 @@
+$border: #dfdfdf;
+$footer: #f6f6f6;
+
+@mixin section($grow, $centered) {
+  flex: #{$grow} 1 0;
+  padding: 0 ru(1.25);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: relative;
+
+  @if $centered {
+    align-items: center;
+  }
+}
+
+.policy-card {
+  border: 1px solid $border;
+  border-radius: ru(.25);
+  max-width: rem-calc(1060px);
+}
+
+.body {
+  padding: ru(1) 0;
+  display: flex;
+}
+
+.footer {
+  background-color: $footer;
+  border-top: 1px solid $border;
+  box-shadow: inset 0 0 10px rgba(0, 0, 0, .06);
+  padding: ru(.5) ru(1.25);
+}
+
+.divider {
+  height: ru(3.25);
+  border-left: 1px solid $border;
+  position: absolute;
+  right: 0;
+}
+
+.node {
+  display: inline-block;
+  @include typography(7);
+  color: color('neutral-3');
+  padding-right: ru(.5);
+}
+
+.policy-type > strong {
+  display: block;
+}
+
+@media #{$small-only} {
+  .body {
+    flex-direction: column;
+  }
+
+  .continue {
+    margin-bottom: ru(.75);
+  }
+
+  .footer {
+    text-align: center;
+  }
+
+  .divider {
+    display: none;
+  }
+
+  .actions, {
+    padding: ru(1) ru(1) 0 ru(1);
+    text-align: center;
+    border-top: 1px solid $border;
+  }
+
+  .policy-type {
+    padding: ru(1);
+    text-align: center;
+  }
+
+  .carrier-logo {
+    text-align: center;
+  }
+
+  .policy-name {
+    max-width: 140px;
+    margin: auto;
+    margin-bottom: ru(.5);
+    line-height: ru(1);
+  }
+}
+
+@media #{$medium-up} {
+  .compare {
+    @include section(2, true);
```

Comment:
@cisacke you can probably simplify these `flex-grow` values.

it seems like `compare` and `carrier-logo` are same width while `type`, `info` and `actions` are the same. You could probably just have `flex-grow: 0` for the first two and then `flex-grow:1` for the last 3. this still ensures that the leftover space is distributed proportionally between these two sets of items as well as evenly within each set.

Reply:
👍 good call, I made that change